tools/travis/build.sh

```diff
@@ -21,7 +21,7 @@ $ANSIBLE_CMD apigateway.yml
 
 cd $ROOTDIR
 
-./gradlew distDocker -x :core:swift3Action:distDocker -x :core:pythonAction:distDocker -PdockerImagePrefix=testing
```

Comment:
Why you remove this?
This excludes is to make the Travis build end below the 50 minutes max.
This related to the gradle task testLean, where swift3 and python are excluded 
https://github.com/openwhisk/openwhisk/blob/master/tests/build.gradle#L18